src/components/signup/SignupState.js

```diff
@@ -401,6 +398,12 @@ const Signup = ({ navigation, screenProps }: { navigation: any, screenProps: any
       } finally {
         setLoading(false)
       }
+    } else if (nextRoute && nextRoute.key === 'MagicLinkInfo') {
+      const { userStorage } = await ready
+      API.sendMagicLinkByEmail(userStorage.getMagicLink(), newState.fullName)
```

Comment:
```suggestion
      API.sendMagicLinkByEmail(userStorage.getMagicLink())
```

Reply:
No, now it’s not. we can go right away without waiting
![image](https://user-images.githubusercontent.com/8187382/69228603-89982d00-0b8c-11ea-99ee-286b4eed76ef.png)

Reply:
No, now it’s not. we can go right away without waiting

Reply:
@sirpy we don't wait when user/add request ends and go to the next step immediately. so when we send magic link we don't have fullname in mongo yet

Reply:
added waiting completion of saving private data